src/httprpc.cpp

```diff
@@ -90,7 +89,7 @@ static bool multiUserAuthorized(std::string strUserPass)
     if (mapMultiArgs.count("-rpcauth") > 0)
     {
         // Search for multi-user login/pass "rpcauth" from config
-        BOOST_FOREACH (std::string strRPCAuth, mapMultiArgs["-rpcauth"])
+        for (std::string strRPCAuth : mapMultiArgs["-rpcauth"])
         {
```

Comment:
minor, but couldn't we do

- for (std::string strPRCAuth...
+for (std::string &strRPCAuth...